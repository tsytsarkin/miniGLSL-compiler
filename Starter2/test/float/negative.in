{
    float f = -12345.54321;
}
