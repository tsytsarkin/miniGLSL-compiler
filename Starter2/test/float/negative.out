  1: {
  2:     float f = -12345.54321;
  3: }
empty declarations
FLOAT_T type
- unary_op
FLOAT_C expression
unary_op expression
assignment declaration
declarations
empty statements
scope
program
